azure-kusto-ingest/tests/test_descriptors.py

```diff
@@ -47,34 +55,38 @@ def test_zipped_file_with_size(self):
                 assert stream.readable()
             assert stream.tell() == 0
 
-        assert stream.closed == True
+        assert stream.closed is True
 
     def test_gzip_file_without_size(self):
         """Tests FileDescriptor without size and zipped file."""
         filePath = path.join(path.dirname(path.abspath(__file__)), "input", "dataset.csv.gz")
         descriptor = FileDescriptor(filePath, 0)
         with descriptor.open(False) as stream:
-            assert descriptor.size == self.uncompressed_size
+            # TODO: since we don't know if the file is opened on CRLF system or an LF system, allow both sizes
+            #   a more robust approach would be to open the file and check
+            assert descriptor.size in (self.uncompressed_size, self.uncompressed_size_2)
             assert descriptor.stream_name.endswith(".csv.gz")
             if sys.version_info[0] >= 3:
                 assert stream.readable()
             assert stream.tell() == 0
 
-        assert stream.closed == True
+        assert stream.closed is True
 
     def test_zip_file_without_size(self):
         """Tests FileDescriptor without size and zipped file."""
         filePath = path.join(path.dirname(path.abspath(__file__)), "input", "dataset.csv.zip")
         descriptor = FileDescriptor(filePath, 0)
         with descriptor.open(False) as stream:
             # the zip archive contains 2 copies of the source file
-            assert descriptor.size == self.uncompressed_size * 2
+            # TODO: since we don't know if the file is opened on CRLF system or an LF system, allow both sizes
+            #   a more robust approach would be to open the file and check
+            assert descriptor.size in (self.uncompressed_size * 2, self.uncompressed_size_2 * 2)
```

Comment:
I think that this file is not sensitive to the GIT line ending issue, since we check the file meradata for the size.
Has this test failed?

Reply:
nope, sorry i'll revert this